build.gradle

```diff
@@ -258,7 +258,7 @@ dependencies {
     compile(group: 'io.searchbox', name: 'jest', version: '6.3.1') {
         exclude group: 'org.apache.logging.log4j', module: 'log4j-core'
     }
-    compile(group: 'org.elasticsearch', name: 'elasticsearch', version: '6.4.2') {
+    compile(group: 'org.elasticsearch', name: 'elasticsearch', version: '6.8.4') {
```

Comment:
Alex do we need to bump up this? The reason why I'm asking is that the ES backend we use is on version `6.4.2` so ideally it would be better to use a client of corresponding version

Reply:
As per the other PR, there are CVEs for 6.4.2; 6.8.2 was the lowest unaffected version I could find.